app/validators/placeholders/placeholder_validator.py

```diff
@@ -111,42 +111,46 @@ def validate_option_label_from_value_placeholder(self, answer_id):
     def validate_answer_type_for_transform(
         self, argument, argument_name, transform_type
     ):
-        if (
+        if not (
             transform_type in ["format_unit", "format_percentage"]
             and argument_name == "value"
             and argument.get("source") == "answers"
         ):
```

Comment:
Double negative should be avoided.

Reply:
Using "any" here now, is that better? Was there double negation here in the first place?